pkg/cmd/roachtest/test_registry_test.go

```diff
@@ -41,5 +43,16 @@ func TestMakeTestRegistry(t *testing.T) {
 		require.EqualValues(t, 4, s.CPUs)
 		require.True(t, s.TerminateOnMigration)
 	})
+}
+
+func TestPrometheus(t *testing.T) {
+	r, err := makeTestRegistry(spec.AWS, "foo", "zone123", true)
+	require.NoError(t, err)
+
+	f := r.PromFactory()
 
+	rawName := "restore/nodes=4/duration"
+	promName := registry.PromSub(rawName)
+	require.Equal(t, "restore_nodes_4_duration", promName)
+	f.NewGauge(prometheus.GaugeOpts{Namespace: prometheusNameSpace, Name: promName})
```

Comment:
Not sure what this is intended to test. This unit test is basically exercising the helper `PromSub`.

Reply:
Exactly. Added better documentation to reflect this.